src/test/resources/cartridge/testserver-scm-1.rockspec

```diff
@@ -0,0 +1,19 @@
+package = 'testserver'
+version = 'scm-1'
+source  = {
+	url = '/dev/null',
```

Comment:
tab? maybe it's my code problem

Reply:
Yes^ this file is a copy of cluster/testserver-scm-1.rockspec

Reply:
You just can customize IDEA and reformat all code with option + command + L